input/pagecontent/examples.md

```diff
@@ -78,7 +44,7 @@ The patient is two years out from surgery and has undergone surveillance imaging
 * [`teleradiotherapy-prescription-delivery-jenny-m`](Procedure-teleradiotherapy-prescription-delivery-jenny-m.html) ([TeleradiotherapyPrescriptionDelivery])
```

Comment:
the the “([])” presentation renders with these characters in the Extended Example. Is this on purpose or an error in markdown formatting?

Reply:
Left out the "include" needed to pull in the references. Thanks for catching this.